foodhub/accounts/serializers.py

```diff
@@ -0,0 +1,113 @@
+from rest_framework import serializers
+from .models import User
+from django.contrib.auth import authenticate
+from rest_framework.validators import UniqueValidator
+from django.contrib.auth.password_validation import validate_password
+from rest_framework import serializers
+from accounts.validators import (validate_store_merchant,
+                                 validate_items_merchant, validate_total_cost)
+from accounts.models import (Merchant, Item, Store, Order)
+
+
+class RegisterSerializer(serializers.ModelSerializer):
+    email = serializers.EmailField(required=True, validators=[
```

Comment:
Instead of this, u can add `unique=True` on email field itself.

Reply:
As per requirement i have updated my serializers.py file